src/main/java/by/exadel/internship/config/WebSecurityConfig.java

```diff
@@ -69,10 +69,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .authorizeRequests()
-                .antMatchers("/auth/**")
-                .permitAll()
-                .antMatchers("/internships")
-                .permitAll()
+                .antMatchers("/auth/**").permitAll()
+                .antMatchers("/internships", "/forms").permitAll()
```

Comment:
Please
- combine matchers with the same permissions (`permitAll()`) 
- provide constant String[] white_list with endpoints where authorization is not required.
- specify /internships and /forms. InternshipController has endpoints which can't be accessible for not authorized users (like restore intership, get deleted).

Reply:
Done